### PR TITLE
Disable pds appview routes

### PR DESCRIPTION
**Pull request by dholms**
In preperation of merging https://github.com/bluesky-social/atproto/pull/1198 which contains many changes to pds logic including config changes, we'll merge & deploy several piecemeal steps

This PR removes all appview routes & view logic from the PDS.

### CODE REVIEW

File: packages/pds/src/app-view/api/app/bsky/unspecced.ts [hunk 1]
```diff
@@ -132,62 +19,14 @@ export default function (server: Server, ctx: AppContext) {
     auth: ctx.accessVerifier,
     handler: async ({ auth, params }) => {
       const requester = auth.credentials.did
-      const db = ctx.db.db
-      const { limit, cursor, query } = params
-      const { ref } = db.dynamic
-      const feedService = ctx.services.appView.feed(ctx.db)
-
-      let inner = ctx.db.db
-        .selectFrom('feed_generator')
```
Comment (devinivy) [inline packages/pds/src/app-view/api/app/bsky/unspecced.ts:141]: Oh `getPopularFeedGenerators` wasn't proxied yet, huh!

File: packages/pds/src/services/index.ts [hunk 2]
```diff
@@ -74,9 +71,6 @@ export function createServices(resources: {
       imgInvalidator,
     ),
     appView: {
-      actor: ActorService.creator(imgUriBuilder, labelCache),
-      graph: GraphService.creator(imgUriBuilder),
-      feed: FeedService.creator(imgUriBuilder, labelCache),
```
Comment (devinivy) [inline packages/pds/src/services/index.ts:79]: I was wondering if these would all be able to go in this PR.  Awesome 🔥

File: packages/pds/tests/file-uploads.test.ts [hunk 3]
```diff
@@ -5,8 +5,6 @@ import { CloseFn, runTestServer, TestServerInfo } from './_util'
 import { Database, ServerConfig } from '../src'
 import DiskBlobStore from '../src/storage/disk-blobstore'
 import * as uint8arrays from 'uint8arrays'
-import * as image from '../src/image'
```
Comment (devinivy) [inline packages/pds/tests/file-uploads.test.ts:8]: I wonder if some big parts of `src/image` can also be nixed at this point (e.g. at least `src/image/server.ts`?)

**Review by devinivy — APPROVED** [review]
Made it an easy read!  This rocks.  Only really one thing that may need a closer look is the comment re: `describeFeedGenerator`.

File: packages/pds/tests/server.test.ts [hunk 4]
```diff
@@ -85,7 +85,7 @@ describe('server', () => {
     })
   })
 
-  it('compresses large json responses', async () => {
+  it.skip('compresses large json responses', async () => {
```
[the remark this answers is not in the document]
Comment (dholms) [reply packages/pds/tests/server.test.ts:88]: ah right good call

File: packages/pds/src/app-view/api/app/bsky/unspecced.ts (hunk 1 above)
Comment (dholms) [reply packages/pds/src/app-view/api/app/bsky/unspecced.ts:141]: yup lol

File: packages/pds/tests/file-uploads.test.ts (hunk 3 above)
Comment (dholms) [reply packages/pds/tests/file-uploads.test.ts:8]: ripped out all the image server + img uri stuff 👌

File: packages/pds/src/app-view/api/app/bsky/feed/describeFeedGenerator.ts [hunk 5]
```diff
@@ -1,21 +0,0 @@
-import { Server } from '../../../../../lexicon'
-import AppContext from '../../../../../context'
-import { MethodNotImplementedError } from '@atproto/xrpc-server'
-
-export default function (server: Server, ctx: AppContext) {
-  server.app.bsky.feed.describeFeedGenerator(async () => {
```
[the remark this answers is not in the document]
Comment (dholms) [reply packages/pds/src/app-view/api/app/bsky/feed/describeFeedGenerator.ts:6]: Good catch - i totally missed this

Updated the plc document to point to `api.bsky.app` 👍 

https://plc.directory/did:plc:5fllqkujj6kqp5izd5jg7gox/data

File: packages/pds/src/context.ts [hunk 6]
```diff
@@ -46,9 +42,8 @@ export class AppContext {
       labelCache: LabelCache
       runtimeFlags: RuntimeFlags
       backgroundQueue: BackgroundQueue
-      appviewAgent?: AtpAgent
+      appviewAgent: AtpAgent
```
Comment (devinivy) [inline packages/pds/src/context.ts:45]: Not suggesting a change in this PR at all, but in the future I'd be +1 on allowing the PDS to be configured without an appview url, in which case the `app.bsky` endpoints just don't exist on the PDS.

**Review by devinivy — APPROVED** [review]
Massive!! 💎💎

File: packages/pds/src/context.ts (hunk 6 above)
Comment (dholms) [reply packages/pds/src/context.ts:45]: totally agree 👍

**Pull request merged**